subiquity/common/filesystem/labels.py

```diff
@@ -117,6 +117,8 @@ def desc(device):
 def _desc_disk(disk):
     if disk.multipath:
         return _("multipath device")
+    if disk.on_remote_storage():
+        return _("NVMe/TCP drive")
```

Comment:
This looks a bit like an easy thing to forget if we add iscsi or nvme-oF or whatever; I think you should poke under the covers a bit more here.

Reply:
Updated with a default value of "remote drive". Currently no scenario should lead to this value being displayed since only NVMe over TCP drives are considered remote.